dev/bots/test.dart

```diff
@@ -225,8 +118,6 @@ Future<void> main(List<String> args) async {
       } else if (arg.startsWith('--local-engine-src-path=')) {
         localEngineEnv['FLUTTER_LOCAL_ENGINE_SRC_PATH'] = arg.substring('--local-engine-src-path='.length);
         flutterTestArgs.add(arg);
-      } else if (arg.startsWith('--test-randomize-ordering-seed=')) {
```

Comment:
so we're deprecating this feature, and only allowing it via env var?

Overall, it's confusing to me what the current expected behavior is, and the desired behavior.

Reply:
My intent was to co-locate the logic for shuffleseed into one util method and keep the functionality the same, using the arg option if given, then falling back onto the env var otherwise, as was the original behavior https://github.com/flutter/flutter/pull/146592/files#diff-0c28a0a1360db88b9554f7317b5680a2696f5274d4c6e44bad2e23aaa3a4810fR311

Reply:
> keep the functionality the same, using the arg option if given

In this PR, where do we check if the arg option is provided?

Reply:
yeah, this code is pretty inscrutable :)